### ISSUE TITLE
Nullable int parameter validation incorrect "Value must be an integer" 

### ISSUE THREAD

**Issue by gohf** [swagger-api/swagger-ui#6972]
When a nullable integer is expected Swagger-UI will not mark it as invalid if the user just presses execute. However, if the user puts any text in the field and subsequently removes it Swagger-UI will mark the field as invalid.

In the example offset and size are supposed to be optional paging parameters. Notice Offset no longer passes validation whereas size, which has not had a value inputted and then cleared is fine.

### Q&A (please complete the following information)
 - OS: Windows 10
 - Browser: chrome
 - Version: 88.0.4324.150
 - Method of installation: Nuget
 - Swagger-UI version: 3.42.0
 - Swagger/OpenAPI version: OpenAPI 3.0

### Content & configuration
Example Swagger/OpenAPI definition:
```yaml
parameters:
  - name: Offset
    in: query
    schema:
      type: integer
      format: int32
      nullable: true
  - name: Size
    in: query
    schema:
      type: integer
      format: int32
      nullable: true
```

### To reproduce...

1) Load the example OpenApi specification.

![Screenshot 2021-02-19 130313](https://user-images.githubusercontent.com/1460721/108507875-df3d1700-72b2-11eb-8a77-6bf477a684a9.png)

2) Type something into the offset input box

![Screenshot 2021-02-19 130414](https://user-images.githubusercontent.com/1460721/108507959-fed43f80-72b2-11eb-8fd0-f8f869a279be.png)

3)  Delete the text in the offset input box

![Screenshot 2021-02-19 130535](https://user-images.githubusercontent.com/1460721/108508089-30e5a180-72b3-11eb-975d-86360b0de0ab.png)

4) Press Execute

![Screenshot 2021-02-19 130637](https://user-images.githubusercontent.com/1460721/108508179-51adf700-72b3-11eb-8f73-ce499f21731b.png)

### Expected behavior
Validation should not invalidate.

**Comment by mathis-m** [comment]
This was fixed with [v3.43.0](https://github.com/swagger-api/swagger-ui/releases/tag/v3.43.0).
Duplicate of #6927.

**Comment by gohf** [comment]
Marvellous, many thanks.
